pycue/opencue/wrappers/group.py

```diff
@@ -197,13 +197,13 @@ def pendingJobs(self):
         :return: total number of running jobs"""
         return self.data.group_stats.pending_jobs
 
-    def hasParent(self):
-        """Whether this Group/NestedGroup has a parent group.
+    def isRootGroup(self):
+        """Whether this Group/NestedGroup is a root group.
 
         :rtype: bool
-        :return: whether the group has a parent
+        :return: whether the group is a root group.
         """
-        return self.data.HasField('parent')
+        return hasattr(self.data, 'parent') and not self.data.HasField('parent')
```

Comment:
Based on my read of the code, only `NestedGroup` can be root group, so I think it's safe to refactor this to be a little more clear. Basically, is this a `NestedGroup` and is the parent field empty.

Reply:
Yeah, that's my read of it too. Ok, I've reorganized the code here and the type checking to match.